diff --git a/kernel/nvidia-drm/nvidia-drm-drv.c b/kernel/nvidia-drm/nvidia-drm-drv.c
index e48378f..5b5ed2b 100644
--- a/kernel/nvidia-drm/nvidia-drm-drv.c
+++ b/kernel/nvidia-drm/nvidia-drm-drv.c
@@ -35,6 +35,7 @@
 #include "nvidia-drm-gem-nvkms-memory.h"
 #include "nvidia-drm-gem-user-memory.h"
 #include "nvidia-drm-gem-dma-buf.h"
+#include <linux/version.h>

 #if defined(NV_DRM_AVAILABLE)

@@ -204,7 +204,9 @@ static const struct drm_mode_config_funcs nv_mode_config_funcs = {
     .atomic_check  = nv_drm_atomic_check,
     .atomic_commit = nv_drm_atomic_commit,
 
+#if LINUX_VERSION_CODE < KERNEL_VERSION(6,12,0)
     .output_poll_changed = nv_drm_output_poll_changed,
+#endif
 };
 
 static void nv_drm_event_callback(const struct NvKmsKapiEvent *event)
@@ -296,6 +296,10 @@ static void nv_drm_enumerate_encoders_and_connectors
             "Failed to enumurate NvKmsKapiDisplay count");
     }
 
+    NV_DRM_DEV_LOG_INFO(
+            nv_dev,
+            "nv_drm_enumerate_encoders_and_connectors nDisplays=%u", nDisplays);
+
     if (nDisplays != 0) {
         NvKmsKapiDisplay *hDisplays =
             nv_drm_calloc(nDisplays, sizeof(*hDisplays));

